tigerpath/majors_and_certificates/majors/CEE.json

```diff
@@ -681,20 +699,30 @@
               "min_needed": null,
               "explanation": null,
               "course_list": [
-                "CEE 205: Mechanics of Solids",
-                "CEE 262A: Structures and the Urban Environment",
+                "CEE 262B: Structures and the Urban Environment",
                 "CEE 312: Statics of Structures",
                 "CEE 361: Matrix Structural Analysis and Introduction to Finite-Element Methods"
               ]
             },
+            {
+              "name": null,
+              "max_counted": 1,
+              "min_needed": null,
+              "explanation": null,
+              "course_list": [
+                "CEE 205: Mechanics of Solids",
+                "MAE 223 Modern Solid Mechanics"
```

Comment:
there should be a `:` after `MAE 223` (fix this in other places too)

Reply:
Fixed